subsys/net/l2/openthread/Kconfig

```diff
@@ -129,6 +129,15 @@ config OPENTHREAD_MTD
 	bool "MTD - Minimal Thread Device"
 endchoice
 
+config OPENTHREAD_MTD_SED
+	bool "SED - Sleepy End Device"
+	depends on OPENTHREAD_MTD
+
+config OPENTHREAD_POLL_PERIOD
+	int "Poll period for sleepy end devices [ms]"
+	default 236000
```

Comment:
236 seconds? Is it good default? (Might be good value for *some* production devices (hmm, I personally wouldn't like to own a temperature sensor which takes 4mins to update), but unlikely a good default.

Reply:
Thread specification defaults the child timeout to 240 seconds. A sleepy end device needs to poll parent within this period, otherwise, it will be dropped. 4 extra seconds are for possible retry attempts. It's the same default as we use in OpenThread.

And just to clarify - the temperature sensor scenario you brought up is not a good example of how sleepy end devices poll mechanism work. A sleepy end device polls the parent (powered router) periodically, to obtain any data the parent queued for it. There are no restrictions on when a sleepy child can send data to its parent - it can do it as often as it wants (e.g. when temperature value changes above some configured threshold). Of course, if the sleepy end device expects a response for its notification (for instance ACK on CoAP), it should either decrease the poll period or switch to MTD mode (non-sleepy) until response is recevied.

Reply:
@rlubos, Thanks for explanation. So, it's effectively more like a timeout than a period, though OpenThread terminology prefers the latter.